Music/spotify.10s.sh

```diff
@@ -4,15 +4,16 @@
 #
 # by Jason Tokoph (jason@tokoph.net)
 #    Marcin Swieczkowski (scatman@bu.edu)
+#    Benji Encalada Mora (@benjifs)
 #
 # Shows current track information for Spotify
 
 # metadata
 # <bitbar.title>Spotify Now Playing</bitbar.title>
-# <bitbar.version>v1.2</bitbar.version>
-# <bitbar.author>Marcin S., Jason Tokoph</bitbar.author>
+# <bitbar.version>v1.3</bitbar.version>
+# <bitbar.author>Benji Encalada Mora, Marcin S., Jason Tokoph</bitbar.author>
```

Comment:
Hey @benjifs - we try and put the original author first in the list if you don't mind :)

Reply:
Sorry about the delay. Should be fixed now